libs/sdk-ui-kit/src/AppearancePage/index.ts

```diff
@@ -0,0 +1,20 @@
+// (C) 2022 GoodData Corporation
```

Comment:
I would rename this package to `SettingWidget`. I would then put under it a sub directory called `StylingSettingWidget` and put there current content of `StylingPicker` and rename it to `StylingSettingWidget` (I would rename its sub components as well). Then I would create another sub directory called `SimpleSettingsWidget` and put there a current `WhiteLabeling` component and rename it to `SimpleSettingWidget`. 

Then there could be way less components exported out as most of these are internals of the both setting widget variants. You can just export `StylingSettingWidget` and `SimpleSettingsWidget` with their props interfaces. This will be way more clean.